boards/nrf52dk/Makefile.include

```diff
@@ -14,7 +14,7 @@ export JLINK_DEVICE := nrf52
 
 # special options when using SoftDevice
 ifneq (,$(filter nordic_softdevice_ble,$(USEPKG)))
-export JLINK_PRE_FLASH := loadfile $(BINDIR)/softdevice.hex
+export JLINK_PRE_FLASH := erase\nloadfile $(BINDIR)/softdevice.hex
```

Comment:
Like this, on my system, it leads to

```erase\nloadfile <path>/softdevice.hex```

being the first line of burn.seg, which results in an error message.